os-sso/added/openshift-launch.sh

```diff
@@ -70,8 +70,18 @@ JAVA_PROXY_ARGS="$(proxy_options)"
 
 echo "Running $JBOSS_IMAGE_NAME image, version $JBOSS_IMAGE_VERSION"
 
+# TERM signal handler
+function clean_shutdown() {
+  echo "*** JBossAS wrapper process ($$) received TERM signal ***"
+  $JBOSS_HOME/bin/jboss-cli.sh -c ":shutdown(timeout=60)"
+  wait $!
+}
+
+trap "clean_shutdown" TERM
```

Comment:
The changes for EAP had a variable to disable this.  Why the difference between the two?